app/src/pages/Play/index.tsx

```diff
@@ -119,6 +119,29 @@ function Play() {
     return null
   }
 
+  const numCompletedCards = completedCardIds.length
```

Comment:
a lot of this is copied from below -- needed to put up here so I could pass values into `YourTurnContent`.